pkg/runner/local_docker.go

```diff
@@ -297,6 +299,11 @@ func (r *LocalDockerRunner) Run(ctx context.Context, input *api.RunInput, ow *rp
 		return
 	}
 
+	if cfg.Disabled {
+		err = ErrRunnerDisabled
+		return
+	}
+
```

Comment:
If we're doing the check here, is it possible for a user to send a custom `input.RunnerConfig` and overrides the flag? (see the `mergo.Merge` line above)

Could we check the flag higher up in the stack? Maybe in the `supervisor.go` module instead. It's a generic flag and it'd be safer if future maintainers don't have to remember to support this flag.

Reply:
> If we're doing the check here, is it possible for a user to send a custom input.RunnerConfig and overrides the flag? 
You're right, it's possible! The merge will overwrite the default (`env.toml`) config.

> Could we check the flag higher up in the stack? Maybe in the supervisor.go module instead. 
This is the right way to go about it, though the config is a generic map[string]interface, which might not be ideal to check. I'll try to find a way to keep this check clean.